specification/common/transactions/RegisterDelegateTransaction.yaml

```diff
@@ -18,7 +18,13 @@ allOf:
         type: "null"
         description: There is no recipient for this type of transaction
       asset:
-        $ref: "./assets/RegisterDelegateAsset.yaml"
+        allOf:
+          -  $ref: "./assets/RegisterDelegateAsset.yaml"
+          - type: object
+            properties:
+              address:
+                $ref: "../properties/AdamantAddress.yaml"
```

Comment:
Remove this part and describe `address` property inside the `RegisterDelegateAsset.yaml`

Reply:
so you want me to unlink `RegisterDelegateAsset` from `register/RegisterNewDelegateTransaction`? because the 2nd has no `address` property.

Reply:
Yes, `RegisterDelegateAsset` and other assets describes the final state inside the blockchain.

But the `register/RegisterNewDelegateTransaction` as a request body, it should describe his own asset (without `address` property included)